velox/docs/functions/spark/comparison.rst

```diff
@@ -10,7 +10,7 @@ Comparison Functions
 
 .. spark:function:: equalnullsafe(x, y) -> boolean
 
-    Returns true if x is equal to y. Supports all scalar types. The
+    Returns true if x is equal to y. Supports all types. The
```

Comment:
It would be better to add some documentation for the null behavior of complex types.

Reply:
The only difference is related to the nested nulls.
Added, thanks,